ipn/ipnlocal/local.go

```diff
@@ -4791,3 +4798,30 @@ func (b *LocalBackend) ResetAuth() error {
 	}
 	return b.resetForProfileChangeLockedOnEntry()
 }
+
+// StreamDebugCapture writes a pcap stream of packets traversing
+// tailscaled to the provided response writer.
+func (b *LocalBackend) StreamDebugCapture(w http.ResponseWriter) error {
+	var (
+		s   *capture.Sink
+		err error
+	)
+
+	b.mu.Lock()
+	if b.debugSink == nil {
+		if s, err = capture.New(); err != nil {
+			b.mu.Unlock()
+			return err
```

Comment:
have this do the 200 write so the error path can return a nice 500 + error text?

also do a http Flush after the 200 so the HTTP response headers get sent immediately even if no packets are coming?